moveit_ros/moveit_servo/src/pose_tracking.cpp

```diff
@@ -225,35 +223,35 @@ bool PoseTracking::satisfiesPoseTolerance(const Eigen::Vector3d& positional_tole
           (std::abs(z_error) < positional_tolerance(2)) && (std::abs(angular_error_) < angular_tolerance));
 }
 
-void PoseTracking::targetPoseCallback(const geometry_msgs::PoseStampedConstPtr& msg)
+void PoseTracking::targetPoseCallback(const geometry_msgs::msg::PoseStamped::ConstSharedPtr msg)
 {
   std::lock_guard<std::mutex> lock(target_pose_mtx_);
   target_pose_ = *msg;
-
   // If the target pose is not defined in planning frame, transform the target pose.
   if (target_pose_.header.frame_id != planning_frame_)
   {
     try
     {
-      geometry_msgs::TransformStamped target_to_planning_frame = transform_buffer_.lookupTransform(
-          planning_frame_, target_pose_.header.frame_id, ros::Time(0), ros::Duration(0.1));
+      geometry_msgs::msg::TransformStamped target_to_planning_frame = transform_buffer_.lookupTransform(
+          planning_frame_, target_pose_.header.frame_id, rclcpp::Time(0), rclcpp::Duration(0));
```

Comment:
Here is the documentation for `rclcpp::Duration`.  This should be some value other than 0.  It is no longer constructed with a double but is either (seconds, nanoseconds) or (nanoseconds).

Reply:
I'd change this to `0.1s` using chrono literals

Reply:
or `100ms`